src/main/java/no/difi/sdp/client2/domain/kvittering/Kvitteringsinfo.java

```diff
@@ -0,0 +1,74 @@
+package no.difi.sdp.client2.domain.kvittering;
+
+import java.time.Instant;
+
+public class Kvitteringsinfo {
+
+    private String konversasjonsId;
```

Comment:
Mest en kommentar til ettertanke: Kunne `konversasjonsId` og `referanseTilMeldingsId` vært egne objekter? Strenger er jo på en måte en slags primitiver som vi sender rundt, og de holder på like kontekst om _hva_ de er. Tror ikke vi skal gjøre om på det i forbindelse med dette PRet, men har blitt mer og mer fan av å lage enkle objekter for verdier som sendes rundt.